.travis.yml

```diff
@@ -60,15 +60,17 @@ install:
       fi;
     - |
       if [ "$DISTRIB" == "ubuntu" ]; then
-        # Use a separate virtual environment that the one provided by
+        # Use a separate virtual environment than the one provided by
         # Travis because it contains numpy and we want to use numpy
         # from apt-get
         deactivate
         virtualenv --system-site-packages testvenv
         source testvenv/bin/activate
+        pip install -U requests[security]  # ensure SSL certificate works
         pip install -r requirements.txt
-        pip install seaborn sphinx==1.4.9 pytest pytest-cov
+        pip install seaborn sphinx>=1.5 pytest pytest-cov
```

Comment:
We should put in the changelog that support for Sphinx 1.4 is dropped, maybe in a separate PR. This pre 1.5 test was included in Sphinx-Gallery 0.1.6 a year ago because during the transition a lot of things broke. We should probably still test Sphinx<1.6 here.

Reply:
cool, I can mention this in changelog. Can I set this to 1.5.X by just doing `==1.5`?

Reply:
> cool, I can mention this in changelog. Can I set this to 1.5.X by just doing ==1.5?

With pip `==1.5` will install 1.5.0 which is fine for our purposes. Use `pip install 'sphinx==1.5.*'` if you want to install the latest 1.5.X release.